### ISSUE TITLE
[Visual Requirements – Make Code – Instructions] Ensures every HTML document has a lang attribute

### ISSUE THREAD

**Issue by Sheila111** [microsoft/pxt-microbit#3999]
**Describe the bug**
[Visual Requirements – Make Code – Instructions] Ensures every HTML document has a lang attribute

**Steps to reproduce the behavior**
1.Navigate to https://makecode.microbit.org/beta#
2.Radio Games----->Fireflies---->Show Instructions
3.Use Accessibility Insights tool and click FastPass

**Actual behavior**
[Visual Requirements – Make Code – Instructions] Ensures every HTML document has a lang attribute
![image](https://user-images.githubusercontent.com/59192069/115839783-97bb3e80-a44d-11eb-8c0d-9e7168e47e71.png)

**Additional context**
1.OS: Windows(rs6)
2.makecode.microbit.org version: 3.1.75
3.Microsoft MakeCode version: 6.13.18

**Comment by Sheila111** [comment]
**Title**: WCAG 3.1.1: Ensures every HTML document has a lang attribute (html)
**Tags**: Accessibility, WCAG 3.1.1, html-has-lang
**Issue**: Ensures every HTML document has a lang attribute (html-has-lang - https://accessibilityinsights.io/info-examples/web/html-has-lang)
**Target application:** Fireflies - https://makecode.microbit.org/projects/fireflies
**Element path**: html
**Snippet**: <html lang="" xmlns="http://www.w3.org/1999/xhtml" xmlns:og="http://ogp.me/ns#" xmlns:fb="http://www.facebook.com/2008/fbml">
**How to fix**: 
Fix any of the following:
  The <html> element does not have a lang attribute
**Environment**: Chrome version 90.0.4430.85

====

This accessibility issue was found using Accessibility Insights for Web 2.25.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
